src/main/java/house/inksoftware/systemtest/domain/config/infra/InfrastructureLauncher.java

```diff
@@ -49,7 +49,7 @@ public SystemTestConfiguration launchAllInfra(EmbeddedKafkaBroker kafkaBroker,
             } else if (key.equals("mockedServer")) {
                 String path = (String) ((LinkedHashMap) value).get("path");
                 Object warmupSeconds = ((LinkedHashMap) value).get("warmupSeconds");
-                launchMockedServer(path, warmupSeconds == null ? MOCKED_SERVER_DEFAULT_WARMUP_SECONDS : (int) warmupSeconds);
+                launchMockedServer(path, warmupSeconds == null ? MOCKED_SERVER_DEFAULT_WARMUP_SECONDS : (int) warmupSeconds, restTemplate);
```

Comment:
if we can reliably detect that container is up and running there's no need  for MOCKED_SERVER_DEFAULT_WARMUP_SECONDS concept